include/gepetto/gui/mainwindow.hh

```diff
@@ -157,6 +157,9 @@ signals:
 
         /// \}
 
+      void registerShortcut(QString widgetName, QString actionName, QAction* action);
+      void registerShortcut(QString widgetName, QAction* action);
```

Comment:
To keep the documentation consistent, could you put that in the section `available_in_python` ?